topic/machine-learning/llm-langchain/cratedb_rag_customer_support_langchain.ipynb

```diff
@@ -86,7 +86,7 @@
     "#!pip install -r requirements.txt\n",
     "\n",
     "# Note: If you are running in an environment like Google Colab, please use the absolute path of the requirements:\n",
-    "#!pip install -r https://raw.githubusercontent.com/crate/cratedb-examples/main/topic/machine-learning/llm-langchain/requirements.txt\""
+    "#!pip install -r https://raw.githubusercontent.com/crate/cratedb-examples/main/topic/machine-learning/llm-langchain/requirements.txt"
```

Comment:
Please only submit this improvement, and omit the others. Dependencies have been improved on behalf of those patches:

- GH-425
- GH-428

Reply:
Undid the dependency changes